addons/controller_icons/objects/ControllerIconTexture.gd

```diff
@@ -68,6 +68,22 @@ enum ShowMode {
 		show_mode = _show_mode
 		_load_texture_path()
 
+
+## Forces the icon to show a specific controller style, regardless of the
+## currently used controller type.
+##[br][br]
+## This will override force_device if set to a value other than NONE.
+##[br][br]
+## This is only relevant for paths using input actions, and has no effect on
+## other scenarios.
+
+
+
+@export var force_controller_icon_style: ControllerSettings.Devices = ControllerSettings.Devices.NONE:
```

Comment:
There is a lot of whitespace between the docs and the variable, please remove them.